src/nanoemoji/config.py

```diff
@@ -159,6 +162,7 @@ class FontConfig(NamedTuple):
     fea_file: str = "features.fea"
     glyphmap_generator: str = "nanoemoji.write_glyphmap"
     bitmap_resolution: int = 128
+    use_zopflipng: bool = True
```

Comment:
zopfli is kinda slow, should we default to False here?

Reply:
True is probably the right default. Does zopflipng have a quality option? - common in compressors. If so, perhaps this is an int, default is whatever max is, and some value (-1?) turns it off

Reply:
there's no straightforward --quality option I'm afraid. There's the number of iterations to try (there's actually two parameters that control that, one for small files and one for larger files), where more iterations make it slower but provide slightly better compression. noto-emoji builder simply calls it with default parameters, I think it's not worth it to add options to control the iterations, the defaults already seem to be the best compromise.